Docs/input.html

```diff
@@ -0,0 +1,268 @@
+                <a href="#">
+                  Colors
+                </a>
+              </li>
+            </ul>
+            <li class="m-05">
+              <h6 class="py-1">Components</h6>
+            </li>
+            <ul>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/avatar.html">
+                  Avatar
+                </a>
+              </li>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/alert.html">
+                  Alert
+                </a>
+              </li>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/badge.html">
+                  Badge
+                </a>
+              </li>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/button.html">
+                  Button
+                </a>
+              </li>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/card.html">
+                  Card
+                </a>
+              </li>
+              <li class="side-nav-link my-1 mx-1 px-2 py-05">
+                <a href="../Docs/image.html">
+                  Image
+                </a>
+              </li>
+              <li class="side-nav-link active my-1 mx-1 px-2 py-05">
+                <a href="../Docs/input.html">
+                  Input
+                </a>
+              </li>
+            </ul>
+          </ul>
+        </nav>
+      </aside>
+
+      <!-- main content -->
+      <div class="doc-content m-2">
+        <h1 class="mb-2">Input</h1>
+        <div>
+          <p>
+            Input fields are an essential user interface design element. They
+            allow users to enter the information into the system's interface.
+            AceUI provides text input, checkbox input and radio input.
+          </p>
+        </div>
+
+        <!-- Text Label Input -->
+
+        <section class="doc-section">
+          <h3 class="mb-1">Text Input</h3>
+          <div>
+            <p>
+              AceUI provides text input fields of type - simple, successful and
+              error.
+            </p>
+          </div>
+          <div class="component-container my-2 p-2">
+            <div class="input">
+              <label for="text">Text Input</label>
+              <input
+                type="text"
+                placeholder="Simple text input field"
+                class="input-simple"
+              />
+            </div>
+
+            <div class="input">
+              <label for="text">Username<sup>*</sup></label>
+              <input
+                type="text"
+                placeholder="Valid text input"
+                class="input-success"
+              />
+            </div>
+
+            <div class="input">
+              <label for="text">Password<sup>*</sup></label>
+              <input
+                type="text"
```

Comment:
For password's, type can be changed to password

Reply:
thanks for pointing out, made the changes.

Reply:
can you please approve the PR, if everything looks good.